src/scripts/validation.js

```diff
@@ -0,0 +1,99 @@
+export const validationConfig = {
+  formSelector: ".modal__form",
+  inputSelector: ".modal__input",
+  submitButtonSelector: ".modal__submit-btn",
+  inactiveButtonClass: "modal__submit-btn-disabled",
+  inputErrorClass: "modal__input-type-error",
+  errorClass: "modal__error",
+};
+
+export function resetValidation(formEl, validationConfig) {
+  const inputList = Array.from(
+    formEl.querySelectorAll(validationConfig.inputSelector)
+  );
+  const buttonElement = document.querySelector(
+    validationConfig.submitButtonSelector
+  );
+
+  inputList.forEach((inputElement) => {
+    hideInputError(formEl, inputElement, validationConfig);
+    checkInputValidity(formEl, inputElement, validationConfig);
```

Comment:
`  checkInputValidity(formEl, inputElement, validationConfig);` is not needed inside `resetValidation` because it causes **error messages** under the empty inputs